packages/api-docs-builder/buildApiUtils.ts

```diff
@@ -520,3 +543,198 @@ export const stringifySymbol = (symbol: ts.Symbol, project: TypeScriptProject) =
 
   return formatType(rawType);
 };
+
+export function updateComponentPages() {
```

Comment:
This is the method that generates the new pages (`react-component/index.js` and `react-component/[docsTab]/index.js`). Current this is configured only for MUI Base.